content/blog/2021/09/2021-09-20-election-period-opened.adoc

```diff
@@ -0,0 +1,95 @@
+---
+:layout: post
+:title: "Jenkins Election 2021"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- olblak
+description: >
```

Comment:
We also need opengraph IMHO

Reply:
Indeed it's missing at this stage, I am working on it

Reply:
In case it helps, we've used the ["Jenkins Social Media Covers" slide deck](https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.g778409cba7_0_46) to assemble opengraph images for other blog posts

Reply:
Thank you @MarkEWaite for the suggestion, I created an image based on your slide